services/src/main/java/org/keycloak/broker/oidc/mappers/AbstractClaimMapper.java

```diff
@@ -50,7 +58,7 @@ public static Object getClaimValue(JsonWebToken token, String claim) {
             } else {
                 Object val = jsonObject.get(component);
                 if (!(val instanceof Map)) return null;
-                jsonObject = (Map<String, Object>)val;
+                jsonObject = (Map<String, Object>) val;
```

Comment:
Could you please leave the format as is? The reason is that it makes a review more extensive if we start accepting code formatting changes. Yeah, we don't have a code style, but we're working on it :)